meta-mender-tegra/recipes-mender/tegra-state-scripts/files/redundant-boot-reboot-state-script

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+echo "Starting nvidia mender enter reboot redundant boot steps" >&2
+
+set -e
+num_slots=`nvbootctrl get-number-slots`
+if [ $num_slots != 2 ]; then
+    echo "Enabling A/B update mode using nv_update_engine" >&2
+    nv_update_engine --enable-ab 1>&2
+fi
+
+new_boot_part=`fw_printenv -n mender_boot_part`
+new_mender_boot_part_mountpoint=/tmp/new_mender_boot_part
+echo "Mounting new install partition ${new_boot_part}" >&2
+mkdir -p ${new_mender_boot_part_mountpoint}
+mount /dev/mmcblk0p${new_boot_part} ${new_mender_boot_part_mountpoint}
+mount -oremount,rw /
+cp ${new_mender_boot_part_mountpoint}/opt/ota_package/bl_update_payload_current /opt/ota_package/bl_update_payload_new
+rm /opt/ota_package/bl_update_payload
+ln -s /opt/ota_package/bl_update_payload_new /opt/ota_package/bl_update_payload
+nv_update_engine --install 1>&2
+# Should never get here, nv_update_engine will reboot on its own after successful update
+rc=$?
```

Comment:
If it it is never supposed to get there, I think one can drop the check of return code and simply fail by reaching the code. What do you think?

Reply:
I agree, that's a better solution